A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/VerticalSpeedIndicator.js

```diff
@@ -727,7 +730,7 @@ class Jet_PFD_VerticalSpeedIndicator extends HTMLElement {
                 }
                 if (this.cursorSVGText) {
                     var displaySpeed = Math.floor(vSpeed / 100);
-                    if (Math.abs(displaySpeed) > 0) {
+                    if (Math.abs(displaySpeed) > 2) {//text is displayed above +/-200ft/min only was 0 
```

Comment:
```suggestion
                    if (Math.abs(displaySpeed) > 2) {
```

Reply:
comment left for future reference and prevent questions